lib/practice_exercises.rb

```diff
@@ -1,13 +1,50 @@
 
-# Time Complexity: ?
-# Space Complexity: ?
+# Time Complexity: O(n)
+# Space Complexity: O(1)
 def remove_duplicates(list)
-  raise NotImplementedError, "Not implemented yet"
+  length = list.length
+  i = 0
+  length.times do
+    value_1 = list[i]
+    value_2 = list[i + 1]
+    
+    if value_1 == value_2
+      list.delete_at(i)
```

Comment:
`delete_at` will cause all elements left of the deleted item to shift one index.  So this causes your time complexity to go to O(n<sup>2</sup>)